src/Support/GeometryProxy.php

```diff
@@ -163,11 +163,7 @@ private function determineCoordinateIndices($index, $length)
      */
     protected function exposeRawIfAvailable($argument)
     {
-        if (method_exists($argument, 'getRawGeometry')) {
-            return $argument->getRawGeometry();
-        }
-
-        return $argument;
+        return (method_exists($argument, 'getRawGeometry')) ? $argument->getRawGeometry() : $argument;
```

Comment:
Why make this one line into a ternary? This is less readable to me than splitting it up.